cmd/server.go

```diff
@@ -144,15 +199,30 @@ func initServerCmd() {
 		Run:   produceServerCmdRunFunc(client, serverStopAction),
 	}
 	var removeCmd = &cobra.Command{
-		Use:     "rm [flags] [ID]",
-		Aliases: []string{"remove"},
-		Short:   "Remove server",
-		Long:    `Remove an existing server.`,
-		Args:    cobra.ExactArgs(1),
-		Run:     produceServerCmdRunFunc(client, serverDeleteAction),
+		Use:   "rm [flags] [ID]",
+		Short: "Remove server",
```

Comment:
You have deleted the alias for `"remove"` here. Any particular reason for this? Shouldn't be in this PR.

Reply:
**Ah yes, i thought it might be unnecessary.**
See  https://github.com/gridscale/gscloud/pull/57#discussion_r459083710
Do you think it's better to keep the `remove` alias?
Forgot i have to clean up the history to make it clearer what I did. I'm gonna change it, thanks 👍

Reply:
Yes. This PR is about "Create Server Function" as you wrote in the subject line. Don't do unrelated changes in a PR.